commands/audit/sca/yarn/yarn_test.go

```diff
@@ -22,30 +23,30 @@ func TestParseYarnDependenciesList(t *testing.T) {
 		"pack5@npm:5.0.0":        {Value: "pack5@npm:5.0.0", Details: biutils.YarnDepDetails{Version: "5.0.0", Dependencies: []biutils.YarnDependencyPointer{{Locator: "pack2@npm:2.0.0"}}}},
 	}
 
-	rootXrayId := utils.NpmPackageTypeIdentifier + "@jfrog/pack3:3.0.0"
+	rootXrayId := techutils.Npm.GetPackageTypeId() + "@jfrog/pack3:3.0.0"
 	expectedTree := &xrayUtils.GraphNode{
 		Id: rootXrayId,
 		Nodes: []*xrayUtils.GraphNode{
-			{Id: utils.NpmPackageTypeIdentifier + "pack1:1.0.0",
+			{Id: techutils.Npm.GetPackageTypeId() + "pack1:1.0.0",
 				Nodes: []*xrayUtils.GraphNode{
-					{Id: utils.NpmPackageTypeIdentifier + "pack4:4.0.0",
+					{Id: techutils.Npm.GetPackageTypeId() + "pack4:4.0.0",
 						Nodes: []*xrayUtils.GraphNode{}},
 				}},
-			{Id: utils.NpmPackageTypeIdentifier + "pack2:2.0.0",
+			{Id: techutils.Npm.GetPackageTypeId() + "pack2:2.0.0",
 				Nodes: []*xrayUtils.GraphNode{
-					{Id: utils.NpmPackageTypeIdentifier + "pack4:4.0.0",
+					{Id: techutils.Npm.GetPackageTypeId() + "pack4:4.0.0",
 						Nodes: []*xrayUtils.GraphNode{}},
-					{Id: utils.NpmPackageTypeIdentifier + "pack5:5.0.0",
+					{Id: techutils.Npm.GetPackageTypeId() + "pack5:5.0.0",
 						Nodes: []*xrayUtils.GraphNode{}},
 				}},
 		},
 	}
 	expectedUniqueDeps := []string{
-		utils.NpmPackageTypeIdentifier + "pack1:1.0.0",
-		utils.NpmPackageTypeIdentifier + "pack2:2.0.0",
-		utils.NpmPackageTypeIdentifier + "pack4:4.0.0",
-		utils.NpmPackageTypeIdentifier + "pack5:5.0.0",
-		utils.NpmPackageTypeIdentifier + "@jfrog/pack3:3.0.0",
+		techutils.Npm.GetPackageTypeId() + "pack1:1.0.0",
```

Comment:
We are using the same function call to GetPackageTypeId() 11 times in this function.
Lets put it in a variable at the beginning and reuse the value